resources/assets/js/components/PageListItem.vue

```diff
@@ -23,7 +23,7 @@
 		</span>
 
 		<span ref="name" class="page-list__text" @click="edit">
-			{{ page.path === '/' ? 'Home page' : (page.revision.title || page.slug) }}
+			{{ page.path === '/' ? 'Home page' : (page.title || page.slug) }}
```

Comment:
The title is available as part of the Page data object itself. Shouldn't need to access it via the revision.